home/tests/test_api.py

```diff
@@ -242,7 +242,7 @@ def test_pagination(self):
         self.assertEqual(content["body"]["text"]["value"]["message"], "WA Message 11")
 
     def test_number_of_queries(self):
-        with self.assertNumQueries(14):
+        with self.assertNumQueries(8):
```

Comment:
This reduced due to in-memory cache being enabled for tests. Wagtail does some caching internally, these additional cached queries aren't anything that we do in our views.